can/interfaces/interface.py

```diff
@@ -96,6 +99,9 @@ def __new__(cls, other, channel, *args, **kwargs):
         elif can.rc['interface'] == 'socketcan_native':
             from can.interfaces.socketcan_native import CyclicSendTask as _nativeCyclicSendTask
             cls = _nativeCyclicSendTask
+        #elif can.rc['interface'] == 'remote':
```

Comment:
I started preparing for adding support for the cyclic API but it got too complicated so I put it on hold. I think the proposal made in #80 to have it associated with an already existing bus object would result in a cleaner implementation. I can add some info in this comment and also hide all cyclic API from the documentation until it is fully implemented.